dismob/log.py

```
@@ -122,15 +123,19 @@ def warning(msg: str) -> None:
-    logger.error(msg, stack_info=True, stacklevel=3)
+def error(msg: str, stacktrace: bool = True) -> None:
+    logger.error(msg, stack_info=stacktrace, stacklevel=3)
 
 # --- Discord helpers ---
 
-async def safe_send_message(channel: discord.TextChannel, content: str = None, embed: discord.Embed = None):
+def missing_if_none(value):
+    """Returns MISSING if value is None, else returns the value"""
+    return MISSING if value is None else value
+
+async def safe_send_message(channel: discord.TextChannel, content: str = None, embed: discord.Embed = None, view: discord.ui.View = None, file: discord.File = None) -> discord.Message | None:
```

Ruff (0.14.1)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)